src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java

```diff
@@ -175,6 +182,42 @@ public void completeTask(String taskId,
 
     }
 
+    public List<Task> searchForCompletableTasksUsingEventAndCaseId(SearchEventAndCase searchEventAndCase,
+                                                                   List<PermissionTypes> permissionsRequired,
+                                                                   AccessControlResponse accessControlResponse) {
+
+        //1. Perform the DMN evaluation
```

Comment:
Suggestion: I'd remove these comments. I think the code explains it by itself :) 
Or if you think it's not clear then move the block into a method and choose a convenient name for the method to explain what it does such as: private void evaluateDmn(...)